CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # expect-axe-playwright
 
+## 2.3.0
+
+### Minor Changes
+
+- 4276db6: Allow matcher to accept Axe results object
```

Comment:
Hmm. Should I squash the commits and force push on this PR before referencing the commit SHA here? 

I added this commit SHA manually, but it refers to only one of several commits on this PR branch.

Reply:
The changelog should not be manually edited.  Please add a changeset with the change notes.

https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md#i-am-in-a-single-package-repository

Reply:
Okay, second attempt :) 

I read those instructions yesterday, but I've never used this tool before, so when I ran `yarn changeset` yesterday and it added a file to the `.changesets/` directory, and I saw that there were no other files in that directory, I thought that meant that I did something wrong or that there were more steps I needed to complete. So then I ran `yarn changeset version`, which added the note to the change log and the version bump. 

To satisfy my own curiosity, I'm trying to wrap my mind around how this works. If you have the time, could you read what I've written below and see if I'm on the right track? No worries if not :)

Looking at the git log of the `.changesets` directory, it looks to me that something like the following happens:

1. Create a PR, push commits
2. Add a changeset file to the `.changesets` directory. (Via `yarn changeset` and committing and pushing the result).
3. Squash-merge the commits on the PR. (I assume this is how Changeset knows which SHA to use in the change log. Although I suppose it's probably smart enough to use the SHA of a merge commit.)
4. The GitHub action you've configured (on push to `main`) uses the changesets/action to create a pull request that is essentially the output of `yarn changeset version`.
5. When you merge that PR into main, the changesets/action gets triggered a second time, but this time it runs the `yarn release` command? 

Out of curiosity, how do the GitHub releases get added and/or how are the commits tagged with the version number on GitHub? I'm guessing the Changesets Release Action takes care of all that? Since it has an input called `createGithubReleases`, defined as:

- createGithubReleases - A boolean value to indicate whether to create Github releases after publish or not. Default to true

Reply:
Yep, you're right on track with how changesets works.  And yes, the action creates the GitHub release as well.  It's honestly such an awesome tool for versioning, way superior to the old Lerna flow we used to use for our monorepos before.  The other awesome part is it's all PR based, so you can have strict controls on push access, everything including versioning goes through a pull request, so no one needs direct push access when updating versions and publishing.

Reply:
It looks really neat!

Okay so if I may bother you with another question... 😁 

One thing I wonder, since the changeset file only records the delta of the semantic versioning, meaning it only records whether the set of changes represent major, minor, or patch version upgrades, it seems that the order in which PRs get merged matters.

Scenario 1:

1. PR #23 is merged, package goes from version 2.2.1 to 2.3.0
3. PR #24 is merged, version 2.3.0 to 3.0.0

Scenario 2:

1. PR #24 is merged, package goes from version 2.2.1 to 3.0.0
3. PR #23 is merged, version 3.0.0 to 3.0.1

In my particular case with these two PRs, I think either order is fine, but it seems like a pretty heavy assumption that this is generally the case, especially where major changes are concerned.

Reply:
Order doesn't matter, all changesets will be merged to create the final version using the highest version bump.  This allows notes in the changelog to still be separated by major/minor/patch, but the version bump will be correct.  In this case, regardless of the merge order the new version will be 3.0.0.

Of course, that's assuming I wait to release until this PR is merged, which I'm planning on doing 🙂

Reply:
That makes sense. Neat!

Thank you for answering my questions. 😃